parser_test.go

```diff
@@ -435,6 +435,88 @@ func TestParser_ParseUnverified(t *testing.T) {
 	}
 }
 
+var setPaddingTestData = []struct {
+	name          string
+	tokenString   string
```

Comment:
Do you think we can toss in the example JWT token from this comment?

https://github.com/golang-jwt/jwt/issues/92#issuecomment-905442988

Reply:
Just added! Note that I had to add the public key to the repo, added a note where we reference it.